content/docs/details/scaffolder/index.md

```diff
@@ -505,6 +547,19 @@ steps:
         - java
         - ruby
 ```
+#### Outputs
+The `publish:github` action produces two step outputs: `remoteUrl` is the url for the newly created repository; `repoContentsUrl` is a url that shows the contents of the repository.
```

Comment:
I'd say these are better as a list so they can be scanned by the reader instead of having to read through a horizontal text

Reply:
Good idea